python/gui/MaaS/static/common/js/domain.js

```diff
@@ -0,0 +1,206 @@
+
+function titleCase(str) {
+    return str.replaceAll("_", " ").toLowerCase().split(' ').map(function(word) {
+        return word.replace(word[0], word[0].toUpperCase());
+    }).join(' ');
+}
+
+function loadFabricNames() {
+    $.ajax(
+        {
+            url: "fabric/names",
+            type: 'GET',
+            error: function(xhr,status,error) {
+                console.error(error);
+            },
+            success: function(result,status,xhr) {
+                if (result != null) {
+                    result['fabric_names'].forEach(function(name) {
+                        $("#fabric-selector").append("<option value='" + name + "'>" + titleCase(name) + "</option>");
```

Comment:
This can be refactored to look a little more like `$("#fabric-selector").append(`<option value="${name}">${titleCase(name)}</option>`);